libs/core/kiln_ai/adapters/fine_tune/test_vertex_finetune.py

```diff
@@ -276,15 +276,15 @@ async def test_generate_and_upload_jsonl(
     )
 
     mock_path = Path("mock_path.jsonl")
-    expected_uri = "gs://kiln-ai-data/1234567890/mock_path.jsonl"
+    expected_uri = "gs://kiln-ai-data-test-project/1234567890/mock_path.jsonl"
```

Comment:
why change this string? is it only going to work on your account now (and it only worked on mine before?)